### PR TITLE
fix(cargo-doc): fix wrong usage in rust doc string.

### PR DESCRIPTION
**Pull request by BowenXiao1999**
## What's changed and what's your intention?

Otherwise cargo test will fail

## Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests

### CODE REVIEW

File: src/frontend/src/utils/column_index_mapping.rs [hunk 1]
```diff
@@ -135,7 +135,7 @@ impl ColIndexMapping {
     /// # use fixedbitset::FixedBitSet;
     /// # use risingwave_frontend::utils::ColIndexMapping;
     /// let mut remaining_cols = vec![1, 3];
-    /// let mapping = ColIndexMapping::with_remaining_columns(&remaining_cols);
+    /// let mapping = ColIndexMapping::with_remaining_columns(&remaining_cols, 5);
```
Comment (Enter-tainer) [inline src/frontend/src/utils/column_index_mapping.rs:138]:
```suggestion
    /// let mapping = ColIndexMapping::with_remaining_columns(&remaining_cols, 4);
```

**Review by st1page — APPROVED** [review]
LGTM. a little confused why this passed CI, but did not pass `cargo test`. Is there different between cargo test and risedev test？

**Review by Enter-tainer — APPROVED** [review]
nice catch, thx!

**Comment by Enter-tainer** [comment]
> LGTM. a little confused why this passed CI, but did not pass `cargo test`. Is there different between cargo test and risedev test？

Maybe cargo nextest doesn't run doctests?

**Comment by BowenXiao1999** [comment]
> > LGTM. a little confused why this passed CI, but did not pass `cargo test`. Is there different between cargo test and risedev test？
> 
> Maybe cargo nextest doesn't run doctests?

image

indeed. We may need a extra doc test in CI

**Pull request merged**